.devcontainer/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM centos/devtoolset-6-toolchain-centos7:latest
+
+LABEL \
+  description="Ovsrpro Centos 7 Development Container" \
+  name="ovsrpro-dev"
+
+USER 0
+RUN set -eux; \
+  yum --assumeyes update; \
+  yum install --assumeyes \
+    epel-release \
+    # This repository gives us more modern versions of some software, such as Git.
+    https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm; \
+  yum install --assumeyes \
+    cmake3 \
+    git \
+    make \
+    wget; \
+  # yum --assumeyes clean all; \
+  ln -s cmake3 /usr/bin/cmake; \
```

Comment:
Do you also want to add `ln -s ccmake3 /usr/bin/ccmake; `?

Reply:
Sure, I overlooked that one.